app/api/ddi-index-api/dog-breaches.js

```diff
@@ -0,0 +1,40 @@
+const { get, post } = require('./base')
+
+const dogBreachesEndpoint = 'breaches'
+
+const options = {
+  json: true
+}
+
+/**
+ * @typedef BreachCategory
+ * @property {number} id
+ * @property {string} label
+ * @property {string} short_name
+ */
+
+/**
+ * @return {Promise<BreachCategory[]>}
+ */
+const getBreachCategories = async () => {
+  const payload = await get(`${dogBreachesEndpoint}/categories`, options)
+
+  return payload.breachCategories.map(breachCategory => {
```

Comment:
I would personally avoid including functional programming here. It could be quite confusing if people haven't come across it before (like me :) ). Keep is consistent and simple in my view.

Reply:
Actually this isn't, it just looks like it based on the names you've used. Slightly confused - haha

Reply:
changed to
```const [firstLetter, ...restOfLetters] = breachCategory.label
const label = firstLetter.toUpperCase() + restOfLetters.join('')```